tools/buildmgr/cbuildgen/installer/create_installer.sh

```diff
@@ -66,10 +66,18 @@ installer_name=cbuild_install.sh
 mkdir -p ${distdir}/bin
 mkdir -p ${distdir}/etc
 
+cp -R ../bin/. ${distdir}/bin
```

Comment:
I believe this line is not needed (`../bin` does not exist).

Reply:
Ack